Add 2026 COE 1v1 Tournament (#14911)
ppy/osu-wiki: wiki/Tournaments/COE/COE_2026/en.md

The **COE 2026 1v1 Tournament** is a 1v1 single-elimination osu! LAN tournament run entirely in the [osu!(lazer)](/wiki/Client/Release_stream/Lazer) client, and hosted during [cavoe's osu! event](/wiki/Community/cavoe's_osu!_event) 2026 (COE 2026) at Brabanthallen in 's-Hertogenbosch, Netherlands. It is the sixth instalment of the COE tournament series.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration | 2026-06-01/2026-06-13 |
| Online qualifiers | 2026-06-15/2026-04-29 |
| Online stage | 2026-07-11/2026-07-19 |
| Finals mappool showcase | 2026-07-27 (15:30 UTC+2) |
| Finals | 2026-07-30/2026-08-01 |

## Prizes

The COE 2026 1v1 Tournament prizes are yet to be announced.

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 1200€, a Prize from Wooting TBA, COE Merchandise TBA |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 750€, a Prize from Wooting TBA, COE Merchandise TBA |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 500€, a Prize from Wooting TBA, COE Merchandise TBA |
| 4th place | 350€, COE Merchandise TBA |
| 5th-8th place | 250€, COE Merchandise TBA |

> | 9th-16th place | 150€, COE Merchandise TBA |

Reviewer: table cells should use sentence case

```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 1200€, a prize from Wooting TBA, COE merchandise TBA |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 750€, a prize from Wooting TBA, COE merchandise TBA |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 500€, a prize from Wooting TBA, COE merchandise TBA |
| 4th place | 350€, COE merchandise TBA |
| 5th–8th place | 250€, COE merchandise TBA |
| 9th–16th place | 150€, COE merchandise TBA |
```

  3. [PEOPLE 1 - Ratpark feat. Kei Sugawara [Rattus deliceus, Rattus mophinus and Rattus mirandorensis' :ratJam:]](https://osu.ppy.sh/beatmapsets/2439306#osu/5320317)
  4. [xaki - proud-dust [Extra 8.5]](https://osu.ppy.sh/beatmapsets/973222#osu/5354201)
  5. [-45 - Rougoku STRIP [..?]](https://osu.ppy.sh/beatmapsets/2582239#osu/5756979)
- Hidden
  1. [Falcom Sound Team jdk - RED LINE -021- [extra]](https://osu.ppy.sh/beatmapsets/2582256#osu/5757017)
  2. [ginkiha - eastward [forever faithful]](https://osu.ppy.sh/beatmapsets/2582255#osu/5757012)
- Hard Rock
  1. [Hana - tree (Sped Up Ver.) [Triangle-chan's Sakura Spinoff]](https://osu.ppy.sh/beatmapsets/1711023#osu/3496104)
  2. [S2TB Recording - Super Luminal [SHD]](https://osu.ppy.sh/beatmapsets/1620087#osu/3307758)
- Double Time
  1. [cosMo@BousouP - Oceanus [Fortune's Insane]](https://osu.ppy.sh/beatmapsets/267767#osu/618865)
  2. [Function Phantom - Neuronecia [Ethereal]](https://osu.ppy.sh/beatmapsets/186911#osu/541990)
  3. [NOMELON NOLEMON - Dounika Nacchaisou! [hypocrisy]](https://osu.ppy.sh/beatmapsets/2582289#osu/5757116)
- Tiebreaker

>   1. [Camellia feat. Nanahira - finorza [Callionet's Firmament]](https://osu.ppy.sh/beatmapsets/1140744#osu/4647381)

Reviewer: tiebreakers should be bolded

```suggestion
  1. **[Camellia feat. Nanahira - finorza [Callionet's Firmament]](https://osu.ppy.sh/beatmapsets/1140744#osu/4647381)**
```

| **mrekk** ::{ flag=AU }:: | **5** | 1 | ::{ flag=IT }:: kusuriya | [#1](https://osu.ppy.sh/multiplayer/rooms/3610108/events) |
| **MALISZEWSKI** ::{ flag=PL }:: | **5** | 0 | ::{ flag=UA }:: GET LOVED | [#1](https://osu.ppy.sh/multiplayer/rooms/3611764/events) |

Saturday, 18th of July 2026:

| Player 1 |  |  | Player 2 | Match time |
| --: | :-: | :-: | :-- | :-- |
| lystia ::{ flag=GB }:: | 0 | 0 | ::{ flag=FR }:: Hifkil | 13:00 UTC |
| Crestive ::{ flag=US }:: | 0 | 0 | ::{ flag=PL }:: gnahus | 19:00 UTC |

Sunday, 19th of July 2026:

| Player 1 |  |  | Player 2 | Match time |
| --: | :-: | :-: | :-- | :-- |
| lifeline ::{ flag=ID }:: | 0 | 0 | ::{ flag=IT }:: kusuriya | 11:00 UTC |
| Amasetic ::{ flag=FI }:: | 0 | 0 | ::{ flag=SM }:: argweid | 13:00 UTC |
| yary ::{ flag=DE }:: | 0 | 0 | ::{ flag=RO }:: origin_ | 14:00 UTC |
| BoshyMan741 ::{ flag=US }:: | 0 | 0 | ::{ flag=GB }:: fudge | 18:30 UTC |
| xootynator ::{ flag=CA }:: | 0 | 0 | ::{ flag=NL }:: Aheo | 19:30 UTC |

Thursday, 23rd of July 2026:

| Player 1 |  |  | Player 2 | Match time |
| --: | :-: | :-: | :-- | :-- |
| A L E P H ::{ flag=ES }:: | 0 | 0 | ::{ flag=UA }:: GET LOVED | 10:00 UTC |

## Ruleset

> ### 1. General information

Reviewer: numbered headings aren't used anywhere in the wiki

#### 1.1 Range of validity

The COE 2026 1v1 Tournament is operated as part of COE 2026 by Stichting CAVOE EVENTS.

The rules outlined in this rulebook are to be followed by all participants and for all matches played. By participating in the tournament the participant states that they understand and accept all rules.

Stichting CAVOE EVENTS reserves the right to modify this rulebook without notice.

#### 1.2 Participant details

The tournament is split into 3 stages: qualifiers, play-in bracket and final bracket.

> All bracket matches will be direct 1v1 best of 9 matches, except for the Grand Final and 3rd-place matches that will be best of 11. Freeplay and freemod are not enabled throughout the tournament.

Reviewer: all paragraphs need blank lines in between. as is, this will be combined into one line (if that's what you want, delete the single newlines)

#### 1.3 Confidentiality

The content of any correspondence with tournament officials and administrators are deemed confidential. The publication of such material is prohibited for both the participants and tournament officials.

#### 1.4 Broadcast rights

All broadcasting rights of the tournament are owned by Stichting CAVOE EVENTS. This includes but is not limited to, video streams, shoutcast streams, replays, clips and TV broadcasts.

Participants can not refuse to have their matches broadcast, nor can they choose the manner the match will be broadcast.

Stichting CAVOE EVENTS has the right to grant the broadcasting rights to a third party.
Further Stichting CAVOE EVENTS is granting broadcasting rights for single matches to participants if following criteria are met:
The participant is taking part in an online match

> The participant clearly marks that the broadcast is for the tournament (e.g. “COE 2026 1v1” in the title)

Reviewer: quotes should be straight, not curly

```suggestion
The participant clearly marks that the broadcast is for the tournament (e.g. "COE 2026 1v1" in the title)
```

The participant only shows their point of view of the match

If you would like to acquire broadcast rights as a third party you can get in contact as outlined in 1.4

#### 1.5 Contact

> For any questions, notes, or suggestions regarding this rulebook, please contact the tournament organization via [Discord](https://discord.gg/zJ7e6cTVFh)

Reviewer: direct link instead redirect

```suggestion
For any questions, notes, or suggestions regarding this rulebook, please contact the tournament organization via [Discord](https://discord.com/invite/zJ7e6cTVFh)
```

Participants will receive a media schedule beforehand to be informed about the nature, duration and schedule of any activities of this kind that take more than 10 minutes.

Additionally the participants in the grand final and the winner of the match for third place are required to attend the prize ceremony.

#### 1.7 Prize money

The prize money should ideally be paid out 90 days after the tournament has concluded, but may take as long as 180 days for the payment to be completed. If a participant has not provided any payment method to the tournament organization, the payment will be delayed until this is rectified.

Stichting CAVOE EVENTS reserves the right to cancel any pending payment, if any evidence of fraud or foul play has been discovered.

Stichting CAVOE EVENTS is not liable for any taxes or fees the participant may have to deduct from the prize.

Physical and monetary prizes are yet to be determined.

### 2. Requirements

#### 2.1 Game requirements

articipants are not allowed to use custom skin elements to alter core gameplay elements or mechanics. Any modifications to the game not intended by the developer are also strictly forbidden.

> osu!lazer will be used throughout the whole tournament, in the osu! game mode. Participants are expected to always perform updates as prompted by the game.

Reviewer: ```suggestion
osu!(lazer) will be used throughout the whole tournament, in the osu! game mode. Participants are expected to always perform updates as prompted by the game.
```

Should a map end in a draw the beatmap will be replayed, before moving on to the next pick.

### 7. Code of conduct

#### 7.1 Tournament management

Instructions of the referees and the tournament management are to be followed. Decisions labeled as final are not to be objected to.

The tournament management and referees have all rights to apply the rulebook in relation to the tournament and all its matches and give warnings and/or penalties to the participants as they see fit.

#### 7.2 Forbidden behaviors

Participants will be sanctioned if they are violating or attempting to violate any of the following rules:

- Using insulting language and/or gestures, in-game or in person
- Gaining an unfair advantage through any means, including, but not limited to:
  - Cheating software
  - Information abuse
  - Doping
- Showing unsporting behavior
- Faking or being misleading regarding their identity
- Misleading referees
- Not putting in any effort to win a match / Match fixing
- Not following the osu! community guidelines
- Breaking any local laws
- Betting on any matches. This also applies to all staff
- Damaging any equipment that is not your own

#### 7.3 Clothing

> For your own safety you are required to wear closed shoes ( no flip flops or sandals ). We further recommend long trousers. Headwear is allowed.

Reviewer: waht

```suggestion
For your own safety you are required to wear closed shoes (no flip flops or sandals). We further recommend long trousers. Headwear is allowed.
```
